### ISSUE TITLE
error in widgets

### ISSUE THREAD

**Issue by pixelass** [patternplate/patternplate#176]
## Input

    ---
    displayName: "Reference: Using Text Elements"
    description: "A simple overview of Text Elements"
    tags: 
      - Reference
    options:
      order: 8
    ---

    # Text elements

    Dekk provides some basic text elements.

    ## Text

    A simple Text wrapper.

    ## Title

    An h1 title.
    This component is required
    for `@dekk/master-slides~Cover`
    and `@dekk/master-slides~Chapter`

    ```widget
    const React = require("react");
    const {ComponentDemo} = require("@patternplate/widgets");

    module.exports = () => <ComponentDemo id="title" />;
    ```

**text/pattern.json**

```json
{
    "name": "title",
    "version": "1.0.0",
    "tags": [
        "text", 
        "headline",
        "h1",
        "title"
    ],
    "patternplate": {
      "displayName": "Title"
    }
}
```

## Original demo (works)

<img width="723" alt="screen shot 2018-03-13 at 22 06 12" src="https://user-images.githubusercontent.com/1148334/37369902-d46e18de-270a-11e8-88d4-d085ef26a0b6.png">

## Rendered file (text output in the document)

```
TypeError: Cannot read property 'toLowerCase' of undefined
    at a.module.exports.a.renderDOM (/Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:101051:391)
    at a.module.exports.a.render (/Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:101051:228)
    at a.module.exports.a.read (/Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:101050:250)
    at renderToString (/Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:101059:6)
    at /Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:100987:47
    at /Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:103935:5
    at /Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:34756:11
    at done (/Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:35031:19)
    at /Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:35037:7
    at getComponentsForRoute (/Users/greada/Documents/workspace/sinner/dekk/node_modules/@patternplate/cli/lib/start.js:103003:5)
```

## Version

```
node_modules/.bin/patternplate --version
2.0.0-55
```

```json
{
    "@patternplate/cli": "^2.0.0-55",
    "@patternplate/render-styled-components": "^2.0.0-55",
    "@patternplate/widgets": "^2.0.0-55"
}
```

**Comment by marionebl** [comment]
Haven't release the new version with the `ComponentDemo` yet. Use `PatternDemo` and `PatternList` for now, I'll keep around support for them until 3.0

**Comment by pixelass** [comment]
In that case, it was a c&p issue. `PatternDemo` is not documented at all.

**Comment by marionebl** [comment]
Released `2.0.0-56`, which contains the required changes
